tests/openassetio/hostAPI/test_transactions.py

```diff
@@ -59,94 +64,174 @@ def a_scoped_group(a_context):
     return t.ScopedActionGroup(mocked_coordinator, a_context)
 
 
-class TestTransactionCoordinator:
+class Test_TransactionCoordinator_manager:
 
-    # pylint: disable=protected-access
+    def test_when_manager_called_then_returns_the_constructor_supplied_manager(
+            self, mock_manager):
 
-    def test_construction(self, mock_manager):
         coordinator = t.TransactionCoordinator(mock_manager)
         assert coordinator.manager() is mock_manager
 
-    def test_scopedActionGroup(self, transaction_coordinator, a_context):
+
+class Test_TransactionCoordinator_scopedActionGroup:
+
+    def test_when_called_then_returns_ScopedActionGroup_instance(
```

Comment:
[consider] I think the `when_called_then` is a bit redundant in these tests, it seems to be a good case for an exemption from the `when`/`then` rule.

Reply:
Also applies to several other test methods (doing a global search for `test_when_called_then`).

Reply:
A fine point indeed [improved here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/214/commits/144a360a0393ea0adc0e772299d76aaf8f0fa4ba).